src/flb_utils.c

```diff
@@ -169,21 +169,43 @@ void flb_utils_print_setup(struct flb_config *config)
     struct flb_input_plugin *plugin;
     struct flb_input_collector *collector;
     struct flb_input_instance *in;
+    struct flb_filter_instance *f;
+    struct flb_output_instance *out;
 
-    flb_info("Configuration");
+    flb_info("Configuration:");
```

Comment:
I'd suggest `flb_info` -> `flb_debug` unless we know a use case for info logging.

Reply:
Unfortunately that doesn't work; at this point in the execution, `flb_engine_start` has not been called yet, which means `flb_engine_log_start` has not been called yet. This seems to prevent `debug` messages from being shown, even if the log level is debug. Info messages work though. So I am sticking with `flb_info`.